zenoh-flow/src/model/link.rs

```diff
@@ -27,11 +27,11 @@ pub struct ZFLinkDescriptor {
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct ZFFromEndpoint {
```

Comment:
For clarity/coherence purposes, maybe we should rename this struct to `ZFPortFrom`?